app/models/mixins/supports_feature_mixin.rb

```diff
@@ -98,7 +99,7 @@ module SupportsFeatureMixin
     :timeline                   => 'Query for events',
     :update_aggregate           => 'Host Aggregate Update',
     :update                     => 'Update',
-    :update_network_router      => 'Network Router Update',
+    :update_network_router      => 'Network Router Update'
```

Comment:
I was merging with local and just removed it caused it looked trailing to me. Added it back.